AutoRest/Generators/CSharp/CSharp.Unit.Tests/TestExtensions.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Rest.Generator;
+using Microsoft.Rest.Generator.CSharp;
+using Microsoft.Rest.Generator.Extensibility;
+using Microsoft.Rest.Generator.Utilities;
+
+namespace AutoRest.Csharp.Unit.Test
```

Comment:
Could you change this namespace to be `Microsoft.Rest.Generator.CSharp.Unit.Test` so it's consistent with the other projects? (Or I'd prefer `Microsoft.Rest.Generator.CSharp.UnitTest` or even `Microsoft.Rest.Generator.CSharp.Tests`, since it could share a namespace with our other test project)